vars.tf

```diff
@@ -332,4 +342,14 @@ variable "s3_tags" {
   default     = {}
 }
 
+variable "enable_api_gateway" {
```

Comment:
The architecture has a language of internal and external endpoints, of which external endpoints could have a type - S3, API gateway - we're still reasoning about these but I'm unsure if I want an explicit flag that controls adding an api-gateway resource

Reply:
Does that mean the module should not define external endpoints?

Reply:
I feel the module takes a list or dictionary of external endpoints. these external endpoints would contain a type, e.g. api-gateway, S3 etc

What I'm unsure about is if there are that many external endpoints, in that case, saying api_gateway_enabled could make sense

Reply:
It could be, I talked to team who. which they might want to use SQS as an external endpoint.